geo/src/algorithm/simplify_vw.rs

```diff
@@ -250,7 +288,7 @@ where
 fn visvalingam_preserve<T, const INITIAL_MIN: usize, const MIN_POINTS: usize>(
     orig: &LineString<T>,
     epsilon: &T,
-    tree: &mut RTree<Line<T>>,
+    tree: &mut RTree<CachedEnvelope<Line<T>>>,
```

Comment:
I'd be a little surprised if there was much benefit to caching the bounding rect of a line, since there's only ever two points in it, but I'm also not that worried about keeping it in.

Reply:
There was a perf improvement, which I naturally didn't include in the commit 🤦‍♂️

Reply:
Oh, great!